issue_25/get_seq.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Oct 14 11:25:08 2020
+
+@author: Bátora Dániel
+"""
+
+import os 
```

Comment:
There's some guideline on ordering imports:

1. Group all built-in packages (os, typing)
2. Empty line
3. Group all external packages (Bio)
4. Empty line
5. Group all module imports from _this_ package (does not apply here)

Within each group it's useful (though not prescribed) to order imports alphabetically